include/ruby/intern.h

```diff
@@ -686,6 +686,7 @@ void *rb_load_file(const char*);
 void *rb_load_file_str(VALUE);
 /* signal.c */
 VALUE rb_f_kill(int, const VALUE*);
+VALUE rb_f_exists(VALUE, VALUE);
```

Comment:
Do not add exported function unnecessarily.
Implementation functions should be in `internal.h`.